tests/third_party/cupy/math_tests/test_misc.py

```diff
@@ -164,7 +165,7 @@ def test_sqrt(self):
         self.check_unary("sqrt")
 
     @testing.for_all_dtypes(no_complex=True)
-    @testing.numpy_cupy_allclose(atol=1e-5)
+    @testing.numpy_cupy_allclose(atol=1e-5, type_check=has_support_aspect64())
```

Comment:
Should the same `type_check=has_support_aspect64()` used for `check_unary` here (instead of `type_check=False`)?